Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -113,12 +113,8 @@ TEST_CASE(test_bmp)
 
 TEST_CASE(test_gif)
 {
-    // We only support grayscale and non-animated images yet
-    auto bitmap = TRY_OR_FAIL(create_test_rgb_bitmap());
-
-    // Convert bitmap to grayscale
-    for (auto& argb : *bitmap)
-        argb = Color::from_argb(argb).to_grayscale().value();
+    // Let's limit the size of the image so every color can fit in a color table of 256 elements.
+    auto bitmap = TRY_OR_FAIL(create_test_rgb_bitmap({ 16, 16 }));
```

Comment:
instead of changing `create_test_rgb_bitmap()`, do `create_test_rgb_bitmap().cropped({ 16, 16 });` here

Reply:
Sure! I forgot that we have a nice API for that.